File: Pascal/Prog_exe_04.pas
Program Prog_exe_04 ;

Var
	percorrido, litros:integer;

 Begin
 
 	writeln('Programa para o cálculo da média de consumo de um automóvel:');
 	
 	writeln('Digite a distância percorrida com o automóvel (km):');
 	readln(percorrido);
 	
 	writeln('Digite a quantidade de litros de combustível gastos nesta distância:');
 	readln(litros);
 	
 	writeln('A média de consumo é de: ', (percorrido / litros));
  
 End.

File: C/Prog_exe_04.c
#include <stdio.h>

int main()
{
	int percorrido, litros;

	printf("Programa para o cálculo da média de consumo de um automóvel:\n");
	printf("Digite a distância percorrida com o automóvel (km):\n");
	scanf("%d", &percorrido);

	printf("Digite a quantidade de litros de combustível gastos nesta distância:\n");
	scanf("%d", &litros);

	printf("A média de consumo é aprox. de: %d km/L\n", (percorrido / litros));
	return 0;
}